app/assets/sass/patterns/_check-your-answers.scss

```diff
@@ -1,14 +1,79 @@
-.check-your-answers {
 
-  td {
-    @include core-19;
-    vertical-align: top;
+.govuk-check-your-answers {
+
+  > * {
+    position: relative;
+    border-bottom: 1px solid $border-colour;
+
+    > * {
+      display: block;
+    }
   }
 
-  .change-answer {
-    text-align: right;
+  .cya-question {
     font-weight: bold;
-    padding-right: 0;
+    margin: em(12, 19) 4em $gutter/6 0;
```

Comment:
I've changed it now to `em(4, 19)` (not 5 because it was only 5px due to the `$gutter` but should really have been 4px).

I've also rewritten the structure so that the`@media` queries are closer to their mobile equivalents as you mentioned (verbally) earlier.